CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.2.0
+
+* BREAKING CHANGE: Make `YamlMap` preserve parsed key order.
```

Comment:
Maybe flag this as "possibly" breaking? It's not likely to affect most folks, right?

Reply:
It is only mildly breaking in the sense that if anything is broken the fix should be trivial. However, strictly speaking it could break everyone. For example, I found that it breaks some test in the `analyzer`, which everyone depends on. I expect that the change will propagate gradually with minimal disruption.

IOW, done :)

Reply:
We can't land things like this that just break our own code.  It'll prevent internal rolls.

Is someone working with the analyzer team to fix?  That need to happen before landing PRs like this.

fyi - @natebosch

Reply:
I ran internal global presubmit for this change before landing and only found a couple of analyzer test failures that looked easily fixable.

Reply:
Thanks for running the global, but someone needs to fix those before/as we land these.

Reply:
Is there a document listing packages that follow this rule? This seems to pop up randomly. About a year ago `package:quiver` broke `package:build`. There doesn't seem to be a clear way for package authors to be proactive about it (e.g. we didn't even know quiver was a dependency of build).